scout/models/case/case.py

```diff
@@ -38,6 +38,9 @@
     tumor_purity=float,
     tissue_type=str,
     chromograph_images=str,  # path to image files
+    fshd_loci=list(
```

Comment:
What about having these fields defined in the same file as constants (a link to some documentation in a comment would be great), then you use them in here as dict keys? Would be clearer, also for maintenance.

Reply:
I think I'll leave it as is here, but introduce a key-mapping constant in the extension!